src/app/nonprofit/nonprofit-page/nonprofit-page.component.ts

```diff
@@ -35,7 +36,10 @@ export class NonprofitPageComponent implements OnInit {
   ngOnInit(): void {
     this.route.params.pipe(
       concatMap((param) => {
+        // Setting company EIN
         const ein = param['nonprofit-id'];
+        this.nonprofitEIN = (param['nonprofit-id']).toString();
```

Comment:
since we needed to add that nonprofit EIN as a field in our class, we can probably get rid of the const ein variable, and change the usage of const ein to this.nonprofitEIN (I think they should be set at the same time, not sure whether class fields in Angular are asynchronous or not)

Reply:
I tried to get rid of it but I got some errors for some of the code that uses it. I didn't write the code for it so I didn't want to mess with it because I didn't want to mess anything up.